content/swagger/paths.yaml

```diff
@@ -37,6 +37,11 @@
 /api/rest/v1/media-files/{code}/download:
   $ref: ./resources/media_files/routes/media_files_code_download.yaml
 
+/api/rest/v1/job/export/{code}:
+  $ref: ./resources/job/routes/export.yaml
+/api/rest/v1/job/import/{code}:
```

Comment:
I think you should follow the general API guidelines.
Each endpoint directly is in its plural form.
You should use `/jobs/`.